Gemfile

```diff
@@ -8,6 +8,7 @@ gem "config"
 gem "jbuilder", "~> 2.5"
 gem "puma", "~> 3.11"
 gem "rails", "~> 6.0.0.rc1"
+gem "bcrypt", "3.1.13"
```

Comment:
Gem này dùng làm gì em nhỉ?

Reply:
> Gem này dùng làm gì em nhỉ?

BCrypt là một thuật toán mã hóa mật khẩu